requirements-dev.txt

```diff
@@ -12,6 +12,7 @@ flake8-isort==2.7.0
 flake8-string-format==0.2.3
 flake8-tuple==0.4.0
 isort==4.3.21
+pydocstyle==3.0.0
```

Comment:
please add a comment as to why it's pinned low, and when it can be updated

Reply:
I rather add this into the PR which renovates open later on. That PR will fail anyway. And at some point it would be great to pin all deps using Pipenv...

Reply:
Fair enough

Would this be a best practice for our other projects as well? Maybe we should discuss it in the team

Reply:
I think we should get some experience with Pipenv (also in terms of renovate) and once this works well it would be good to discuss to make this the standard (could be done in Caluma as heavily used).

Using pipenv this error just wouldn't have appeared. I open an issue.